toolkit/featureforms/src/main/java/com/arcgismaps/toolkit/featureforms/components/text/FormTextField.kt

```diff
@@ -41,12 +42,24 @@ internal fun FormTextField(
     state: FormTextFieldState,
     modifier: Modifier = Modifier,
 ) {
-    val text by state.value
     val hasError by state.hasError
     val isFocused by state.isFocused
-    val supportingText by state.supportingText
-    val contentLength by state.contentLength
     var clearFocus by remember { mutableStateOf(false) }
+    val isEditable by state.isEditable.collectAsState()
+    val isRequired by state.isRequired.collectAsState()
+    val label by remember(isRequired) {
+        if (isRequired) {
+            mutableStateOf("${state.label} *")
+        } else {
+            mutableStateOf(state.label)
+        }
+    }
```

Comment:
```suggestion
    val label = remember(isRequired) {
        if (isRequired) {
            "${state.label} *"
        } else {
            state.label
        }
    }
```

I think the label need not be a `State` since it's backed by a `State`

Reply:
> I think the label need not be a `State` since it's backed by a `State`

do you mean that since the key's value is delegated to a state variable, there is no need for `label` to be a state variable?